lib/irida/pipelines.rb

```diff
@@ -0,0 +1,129 @@
+# frozen_string_literal: true
+
+require 'json'
+require 'open-uri'
+require 'uri'
+require 'net/http'
+require 'irida/pipeline'
+
+module Irida
+  # Module that reads a workflow config file and registers the available pipelines
+  module Pipelines
+    mattr_accessor :register_pipelines, :find_pipeline_by
```

Comment:
**praise:** the methods should be accesible without using `mattr_accessor` it just means you might have to call them with `Irida::Pipelines.available_pipelines` but I think that is nicer than using the `include` that you did in SubmissionsController.

Reply:
Definitely alot nicer. Cleaned up in d08fd2d05586261bfb12aafc47e7e971a61f711f